.github/workflows/fmt_build_tests.yml

```diff
@@ -28,6 +28,8 @@ jobs:
   unit-and-integration-tests:
     needs: fmt-lint-build
     runs-on: ubuntu-latest
+    env:
+      GCCRS_EXTRA_FLAGS: "-frust-incomplete-and-experimental-compiler-do-not-use"
```

Comment:
I guess it failed because it uses fmt_build_tests.yml from main branch?

Reply:
Mh, that shouldn't be the case. It's also failing locally for me. This might be because the "target option" part of gccrs does not pass the proper env arguments. I'll fix this in this PR.